src/components/settings/ContractVersion/index.tsx

```diff
@@ -42,6 +45,27 @@ export const ContractVersion = ({ isGranted }: { isGranted: boolean }) => {
           Unsupported contract
         </Typography>
       )}
+      <>
+        <br></br>
+        <br></br>
+        {isUpdated! ? (
+          <Typography variant="body1" fontWeight={400}>
```

Comment:
hi, as im using a conditional here, instead of adding margin to both the elements differently how about I warp it inside a div and give it some top-margin.
```suggestion
 <div style={{ marginTop: 15 }}>
        {needsUpdate ? (
          <Typography>✅ Latest version</Typography>
```